app/models/repository_owner/base.rb

```diff
@@ -121,6 +121,13 @@ def self.fetch_org(id_or_login)
       raise NotImplementedError
     end
 
+    def self.sanitize_response_hash(hash)
+      hash
+        .to_hash
```

Comment:
isn't the input already a hash...?

Reply:
each client (Github, Gitlab, Bitbucket) returns their own magic Hash wrapper, so we need to convert it to Hash here before we create the HashWithIndifferentAccess. I'll note this in some docs here to make it more clear for the next person looking at this code.

Reply:
We will also be receiving regular Hashes into this method as well. Calls like https://github.com/librariesio/libraries.io/blob/ef9f9e29ffaad4dc45891fe86b5094d6ecb89ec8/app/models/concerns/github_identity.rb#L86 are already transforming some upstream client output into a Hash and passing it in. I don't think the code needs to change to handle those but wanted to point it out.

Reply:
Can each client (Github, Gitlab, Bitbucket) do the magic hash wrapper -> hash transformation before passing in a hash to the method? Not sure why the method should have to deal with all the different types of specialized objects being passed in when just a plain ol' hash in and hash out would do. From a more technical perspective, it means that whatever gets passed in now implicitly needs to have a `to_hash` method (for better or worse), vs the expectations around if you only trade around Hash instances. 

Or you could name it "sanitize_client_responses" to try and denote that this method trades in specialized types of hashes.

Reply:
> Can each client (Github, Gitlab, Bitbucket) do the magic hash wrapper -> hash transformation before passing in a hash to the method? 

this PR is really just to fix a specific input bug that is really noisy, and I'm trying to avoid changing functionality. If this code becomes more of a priority I think it'd make sense to go back and refactor.